pets.json

```diff
@@ -113,5 +113,25 @@
     "github": "benaadams",
     "instagram": "",
     "twitter": "ben_a_adams"
-  }
+  },
+  {
+    "name": "Queso",
+    "jobTitle": "Management overhead",
+    "img": "https://1drv.ms/u/s!Arurp7yHDAIcqs5de8A5Yul5rvz5dQ",
```

Comment:
Note: I'm not sure how/where we're supposed to host images for this.

Reply:
Yeah, I definitly need to make it easier. I happen to know that the one drive links expire, so i'll pop it in here and link to that 

![image](https://user-images.githubusercontent.com/16246502/120871424-72434980-c550-11eb-9996-6188f78e83ea.png)